content/shmem_collect.tex

```diff
@@ -56,7 +56,7 @@
     \VAR{pSync} must be of type integer and size \CONST{SHMEM\_COLLECT\_SYNC\_SIZE}.
     If you are using \Fortran, it must be a default integer value.  Every element of
     this array must be initialized with the value \CONST{SHMEM\_SYNC\_VALUE} in
-    \CorCpp{} or \CONST{SHMEM\_SYNC\_VALUE} in \Fortran{} before any of the \ac{PE}s
+    \CorCpp{} or \CONST{SHMEM\_SYNC\_VALUE} in \Fortran before any of the \ac{PE}s
```

Comment:
@manjugv There are 114 instances of `\ac{PE}s` in the spec. Should I fix them all?